cmd/cloud/identities/list_test.go

```diff
@@ -77,6 +80,9 @@ func fakeSlugEndpoint(t *testing.T, writer herodot.Writer) *url.URL {
 }
 
 func TestIdentityListNoToken(t *testing.T) {
+	// See https://github.com/ory-corp/cloud/issues/865
```

Comment:
@Demonsthere I disabled the tests for now - we can reenable them once this is merged into cloud